File: src/tests/Markify.Fixtures/Projects/Source/Delegate/DelegateParameters.cs
public delegate void WithoutParameters();

public delegate void WithOneParameter(int foo);

public delegate void WithMultipleParameters(int foo, float bar);

public delegate void WithDefaultParameters(int foo = 1);

public delegate void WithParametersModifiers(ref int foo, out int bar);

public delegate void WithGenericParameters<T>(T foo, T[] bar);

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Delegate/DelegateParameters.vb
Public Delegate Sub WithoutParameters()

Public Delegate Sub WithOneParameter(foo As Integer)

Public Delegate Sub WithMultipleParameters(foo As Integer, bar As Single)

Public Delegate Sub WithDefaultParameters(Optional foo As Integer = 1)

Public Delegate Sub WithParametersModifiers(ByRef foo As Integer, ByVal bar As Single)

Public Delegate Sub WithGenericParameters(Of T)(foo As T, bar As T())